### ISSUE TITLE
Receiving an error when having an Assets field in a column

### ISSUE THREAD

**Issue by ryanshrum** [engram-design/SuperTable#18]
I'm getting an error on entry save (though it may be from Craft) when I have an Assets field as one of my SuperTable columns. I tried a few different file formats and Assets settings and none seem to be working. When I remove the Assets column, it works fine

![dawsu02ge-cduwd3bk5bxvorbp9blqh_0d_sw0iozk8](https://cloud.githubusercontent.com/assets/1153690/8837681/8e528cd6-3095-11e5-9ae6-d95f5d45feb7.png)

**Comment by engram-design** [comment]
@ryanshrum Strange one, I'd hazard a guess this isn't an issue with Super Table as I've done many different fields with various Assets setup. But maybe its something related.

If you have time, might be worth hopping into Slack and chatting there. I'd probably need to check out your log to see whats happening...

Have you tried any other fields in the Super Table field? Particularly Entries or Categories as they're similar element select fields.

**Comment by ryanshrum** [comment]
After chatting with @engram-design in Slack we got to the bottom of it. It is not an issue with Super Table but was an issue with my xDebug settings.

These two links pointed in the right direction:
https://github.com/focuslabllc/craft-field-cheat-sheet/issues/25
http://craftcms.stackexchange.com/questions/1559/why-are-some-entry-detail-views-empty-in-the-control-panel (in addition to seeing the above error, I'd also occasionally see only the section title with no fields — though I thought unrelated)

The default `xdebug.max_nesting_level` is 100. When I upped it to 200, the entry saved fine. Thanks for the help!
